recipes/r-bioseq/meta.yaml

```
@@ -0,0 +1,99 @@
+{% set version = '0.1.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-bioseq
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/bioseq_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/bioseq/bioseq_{{ version }}.tar.gz
+  sha256: b8bb34d9548d9f1f5de8f4ea198e09f656368b6b7a6c56051769b49bf7d12355
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-ape
+    - r-crayon
+    - r-dplyr
+    - r-pillar
+    - r-readr
+    - r-rlang
+    - r-stringdist
+    - r-stringi
+    - r-stringr
+    - r-tibble
+    - r-vctrs
+  run:
+    - r-base
+    - r-ape
+    - r-crayon
+    - r-dplyr
+    - r-pillar
+    - r-readr
+    - r-rlang
+    - r-stringdist
+    - r-stringi
+    - r-stringr
+    - r-tibble
+    - r-vctrs
+
+test:
+  commands:
+    - $R -e "library('bioseq')"           # [not win]
+    - "\"%R%\" -e \"library('bioseq')\""  # [win]
+
+about:
+  home: https://fkeck.github.io/bioseq/
+  license: GPL-2.0-or-later
```

from cran meta data `# License: GPL-3`:
```suggestion
  license: GPL-3.0-only
```